stdlib/public/BackDeployConcurrency/TaskGroup.cpp

```diff
@@ -716,9 +757,11 @@ static void swift_taskGroup_wait_next_throwingImpl(
                          group, waitingTask, polled.retainedTask);
     fillGroupNextResult(context, polled);
     if (auto completedTask = polled.retainedTask) {
-      // it would be null for PollStatus::Empty, then we don't need to release
-      group->detachChild(polled.retainedTask);
-      swift_release(polled.retainedTask);
+      // Remove the child from the task group's running tasks list.
+      _swift_taskGroup_detachChild(asAbstract(group), completedTask);
+
+      // Balance the retain done by enqueueCompletedTask.
```

Comment:
Should be `completedTask` here?

Reply:
No I think this is in reference to the `enqueueCompletedTask` function.

Reply:
Yeah.  Which doesn't exist in the BackDeployConcurrency library, so I should fix the comment.